crates/sovereign-sdk/module-system/sov-modules-api/src/utils.rs

```diff
@@ -7,7 +7,7 @@ pub fn generate_address<C: Context>(key: &str) -> <C as Spec>::Address {
     C::Address::from(hash)
 }
 
-pub fn to_jsonrpsee_error_object(err: impl ToString, message: &str) -> ErrorObjectOwned {
+pub fn to_jsonrpsee_error_object(message: &str, err: impl ToString) -> ErrorObjectOwned {
```

Comment:
Are you sure we can't use some generic reth error instead of 2 args?

Reply:
I will be trying that instead of using this function at all, but for some custom cases we will need this ofc